Documentation/GettingStartedContributing.md

```diff
@@ -0,0 +1,62 @@
+
+This document aims to be a beginner-friendly summary of the existing contribution requirements for Ladybird, remember to read the linked documentation for more information.
+
+## Getting familiar with the project
+Ladybird is a large project making use of many homegrown and third-party libraries and programs, written primarily in C++.
+
+It is recommend you read the FAQs in case it already answers any questions you have, there is also a [Discord server](https://discord.gg/nvfjVJ4Svh), which is the preferred way to ask the maintainers and community for help.
+
+Relevant documentation:
+* [FAQ](FAQ.md)
+* [FAQ on the Ladybird website](https://ladybird.org/#faq)
+* [README](/README.md)
+
+## Building the code
+Ladybird must be built from source during this pre-alpha stage of development, and currently only supports Linux and macOS natively, running it on Windows requires WSL. Please carefully follow the steps outlined in the build instructions.
+
+Relevant documentation:
+* [Build instructions](BuildInstructionsLadybird.md)
+* [Troubleshooting](Troubleshooting.md)
+
+## Finding bugs and other issues
+Here are three suggested ways you can find an issue in Ladybird:
+
+* **By checking the [issue tracker](https://github.com/LadybirdBrowser/ladybird/issues)**
+* **Manually**, by using the browser as you normally would
+* **By finding failing WPT tests** on [WPT.fyi](https://staging.wpt.fyi/results/?label=master&product=ladybird). Note that you do not need to submit seperate issues for these, but fixes are welcome
+
+## Submitting an issue
+If you have found an issue that is not already in the [issue tracker](https://github.com/LadybirdBrowser/ladybird/issues), you may submit it. Do not submit general questions about the project, please use the Discord server instead.
+
+It is recommended you reduce the website to the most minimal amount of HTML/CSS/JS which still results in the error (if applicable), and provide the result expected from other browsers vs Ladybird. Read the detailed issue-reporting guidelines for the exact steps.
+
+Relevant documentation:
+* [Human language policy](/CONTRIBUTING.md#human-language-policy)
+* [Issue policy](/CONTRIBUTING.md#issue-policy)
+* [Detailed issue-reporting guidelines](/ISSUES.md)
+
+## Getting familiar with C++, the code, and submitting new code
+Start small before attempting to make large changes, as they require a better knowledge of C++ and the existing patterns used in the Ladybird codebase.
+
+When submitting a pull request, make sure to follow all requirements linked below. Most importantly:
+* Make correctly formatted, atomic commits (building the project at every commit should succeed)
+* Use the `rebase` function of `git` to keep your local branch up-to-date with master, and to edit existing commits
+* Use the AK library instead of the C++ STL
+* Discuss and resolve any reviews from the maintainers
+
+Relevant documentation:
+* [Full contribution guidelines](/CONTRIBUTING.md)
```

Comment:
We should promote the full contributing.md more as the "but actually read and follow this before submitting your PR", rather than suggesting it as a 'recommended follow-up', as this section of links appears to be at first glance.